static/app/views/settings/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -372,6 +372,13 @@ export class IntegrationListDirectory extends AsyncComponent<
     });
   };
 
+  getCategoryLabel = (value: string) => {
+    if (value === 'api') {
+      return 'API';
+    }
+    return startCase(value);
```

Comment:
nit: No need to change this, but optionally you can use ternary operators for short straight forward if statements like this. One's not better than the other, more of an fyi.
```js
return value === 'api' ? 'API' : startCase(value)
```